tasks/script.json

```diff
@@ -1,4 +1,15 @@
 {
   "description": "This task serves as a bridge between Bolt and Orchestrator and is not intended to be run directly",
-  "input_method": "stdin"
+  "input_method": "stdin",
+  "parameters": {
+    "content": {
+      "type": "String",
+      "description": "The script content to be executed",
+      "sensitive": true
```

Comment:
Are there older versions of the orchestrator that will error on this key?

Reply:
Orchestrator should never have a problem with unexpected keys. Old versions of Bolt can, but they won't be loading the shim task.